src/main/java/OttoResponses.java

```diff
@@ -0,0 +1,27 @@
+public final class OttoResponses {
```

Comment:
I like the use of a class that encapsulates the responses

Reply:
Could look into changing the names of the variables here to screaming snake case, since the otto responses are constant